df.history.xml

```diff
@@ -2515,10 +2515,10 @@
       <int32_t name='entity' ref-target='historical_entity' comment='the guild to which the figure belongs?'/>
       <int32_t name='source_site' ref-target='world_site'/>
       <int32_t name='dest_site' ref-target='world_site'/>
-      <int32_t/>
-      <int32_t/>
-      <int32_t/>
-      <int32_t comment='value of the trade?'/>
+      <int32_t name='unk_v47_1' comment='Somehow connected to traded item, but not mat type/index'/>
+      <int32_t name='unk_v47_2' comment='Somehow connected to traded item, but not mat type/index'/>
+      <int32_t name='unk_v47_3' comment='Somehow connected to traded item, but not mat type/index'/>
+      <int32_t name='profit'/>
```

Comment:
@sedb-dfh identified these in https://github.com/DFHack/df-structures/pull/350/files#diff-35ab45624fa78d4a8c071f3c0fee5d76R2282-R2285

Reply:
Good. I've copied those changes.